docs/source/guide/tutorials/base/demo.rst

```diff
@@ -1,47 +1,65 @@
+a folder of images, a video file, or a web camera id. We can run the demo on PyTorch (.pth) model and IR (.xml) model.
+
+The following line will run the demo on your input source, using PyTorch ``outputs/weights.pth``. 
 
 .. code-block::
 
-    python3 -m venv demo_venv --prompt="demo"
-    source demo_venv/bin/activate
-    python -m pip install -r requirements.txt
+    (demo) ...$ otx demo --input docs/utils/images/wgisd_dataset_sample.jpg \
+                         --load-weights outputs/weights.pth
+
+But if we'll provide a single image the demo processes and renders it quickly, then exits. To continuously visualize inference results on the screen, apply the ``loop`` option, which enforces the processing a single image in a loop.
+
+.. code-block::
 
+    (demo) ...$ otx demo --input docs/utils/images/wgisd_dataset_sample.jpg \
+                         --load-weights outputs/weights.pth --loop
 
-3. The following line will run the demo on your input source, using the model in the ``model`` folder. You can pass as ``input`` a single image, a folder of images, a video file, or a web camera id.
+In this case, you can stop the demo by killing the process in the terminal (``Ctrl+C`` for Linux).
+
+3. In WGISD dataset we have high-resolution images, 
+so the ``--fit-to-size`` parameter would be quite useful. It resizes the resulting image to a specified:
 
 .. code-block::
 
-    (demo) ...$ python3 outputs/deploy/python/demo.py --input docs/utils/images/wgisd_dataset_sample.jpg \
-                                                      --models outputs/deploy/model
+    (demo) ...$ otx demo --input docs/utils/images/wgisd_dataset_sample.jpg \
+                         --load-weights outputs/weights.pth --loop --fit-to-size 800 600
+
+4. If we want to pass an images folder, it's better to specify the delay parameter, that defines, how much millisecond pause will be held between showing the next image.
+For example ``--delay 100`` will make this pause 0.1 ms.
 
-You can press ``Q`` to stop inference during the demo running.
 
-The model inference on your custom image will look like this:
+5. If we want to show inference speed right on images, 
+we can run the following line:
 
-.. image:: ../../../../utils/images/wgisd_pr_sample.jpg
-  :width: 600
-  :alt: this image shows the inference results on the WGISD dataset
+.. code-block::
 
-.. note::
+    (demo) ...$ otx demo --input docs/utils/images/wgisd_dataset_sample.jpg \
+                         --load-weights outputs/weights.pth --loop \
+                         --fit-to-size 800 600 --display-perf
 
-    If you provide a single image as input, the demo processes and renders it quickly, then exits. To continuously
-    visualize inference results on the screen, apply the ``loop`` option, which enforces processing a single image in a loop.
-    In this case, you can stop the demo by killing the process in the terminal (``Ctrl+C`` for Linux).
+.. The result will look like this:
```

Comment:
this ".." in the beginning of the line is intended or not?

Reply:
It's a commented section, because I intended to add this text, but can't do it now, since I wasn't able to run the demo on remote machine :(